bin/injector/Dockerfile

```diff
@@ -14,13 +14,11 @@ FROM gcr.io/distroless/python3-debian11:latest
 # binaries used by the chaos-injector, ran as commmands
 COPY --from=binaries /usr/bin/df /usr/bin/ls /usr/bin/test /usr/bin/
 COPY --from=binaries /usr/sbin/iptables /usr/sbin/
-COPY --from=binaries /sbin/tc /sbin/tc
+COPY --from=binaries /usr/sbin/tc /sbin/tc
 
 # libraries used by above mentioned binaries (mostly GLIBC related)
 COPY --from=binaries /lib/ld-linux-aarch64.so.[1]  /lib/
 COPY --from=binaries /lib64/ld-linux-x86-64.so.[2] /lib64/
-COPY --from=binaries /lib/tc /lib/tc/
```

Comment:
How did you validated it was no longer necessary?
Was it just no longer there?

Reply:
I have the following during the build of injector docker image:
![image](https://github.com/DataDog/chaos-controller/assets/8859832/a89bb731-7a80-4f1d-b5eb-3484b749a4d0)

> Was it just no longer there?

`docker run -ti ubuntu:23.10 bash -c 'apt update && apt install iproute2 -y && which tc && whereis tc'`

- `which tc`: /usr/sbin/tc
- `whereis tc`: tc: /usr/sbin/tc /usr/lib/aarch64-linux-gnu/tc

> How did you validated it was no longer necessary?

I have run a network drop disruption and I don't notice any regression.